docs/Art/index.md

```diff
@@ -0,0 +1,16 @@
+# Basic Information
+[Camp Backgrounds](https://clangen.io/docs/Art/camp-backgrounds/){.md-button}
+[Cat Sprites](https://clangen.io/docs/Art/cat-sprites/){.md-button}
+[Clan Symbols](https://clangen.io/docs/Art/clan_symbols/){.md-button}
+[Patrol Art](https://clangen.io/docs/Art/patrol-sprites/){.md-button}
```

Comment:
Last but not least: please change those links to be in the proper format -> relative links to the .md file. MkDocs will automatically adjust the links.

Reply:
Ah gotcha! Fixed